### DISCUSSION TITLE
How to set switch enlable or dislable?

### DISCUSSION THREAD

**Discussion by yjlc-pc** [beeware/toga#2650] [Q&A]
I want let my four switches follow my two main switches enable or dislable.How to set switch enlable or dislable?

**Comment by freakboy3742** [comment]
If you mean whether the switch is on or off - read the value of one switch with `first_switch.value`; and set the value on the other switch with `second_switch.value = required_value`. You may want to use a  handler on the [`on_change` event](https://toga.readthedocs.io/en/stable/reference/api/widgets/switch.html#toga.Switch.on_change) of the first switch to respond whenever the switch changes value.

If you mean enabling or disabling the switch (i.e., whether the switch *can* be changed at all), use the [`enabled` property](https://toga.readthedocs.io/en/stable/reference/api/widgets/widget.html#toga.Widget.enabled) that exists on all widgets.
